Tests/TestCellEventCoordinator.swift

```diff
@@ -34,7 +34,7 @@ final class TestCellEventCoordinator: UnitTestCase {
         let model = CollectionViewModel(id: "id", sections: [section])
 
         let coordinator = FakeCellEventCoordinator()
-        coordinator.expectationDidSelect = self.expectation()
+        coordinator.expectationDidSelect = self.expectation(name: "did_select")
```

Comment:
For these situations, I prefer being able to not have to provide a name.

Let's make `name` optional, and then simply default to the function name if no `name` is provided.

Reply:
Done.